src/Processors/QtiV2/Out/AssessmentItemBuilder.php

```diff
@@ -98,26 +115,32 @@ private function map(Question $question)
         // Try to use question `reference` as identifier
         // Otherwise, generate an alternative identifier and store the original reference as `label` to be passed in
         $questionReference = $question->get_reference();
-        $interactionIdentifier = Format::isIdentifier($questionReference, false) ? $questionReference : strtoupper($type)  . '_' . StringUtil::generateRandomString(12);
-        if ($interactionIdentifier !== $questionReference) {
-            LogService::log(
-                "The question `reference` ($questionReference) is not a valid identifier. " .
-                "Replaced it with randomly generated `$interactionIdentifier` and stored the original `reference` as `label` attribute"
-            );
-        }
+        $interactionIdentifier = Format::isIdentifier($questionReference, false) ? $questionReference : strtoupper($type) . '_' . StringUtil::generateRandomString(12);
+
+        $interactionIdentifier = 'RESPONSE';
```

Comment:
Just noticed that this is rewriting the interaction identifier that is assigned immediately before it - and I imagine we don't want to have a non-unique identifier. Does this line just need to be removed?

Reply:
Actually earlier the converted qti.xml files were having more then one interaction in case of multiple questions in single item but now we have changed the flow and we are creating a separate xml file for each question in item json. So there would be only one interaction in a qti file and it is not an issue to have this as an interaction identifier.

Reply:
Ok, so my understanding of what you're saying here is that this line intentionally overrides the original behavior, which used the question reference to build the identifier.

If that's the case, can we remove the old logic? Having both here is confusing and redundant. Also update (or remove) any inline comments or docblock comments that describe the old behavior.